soc/nordic/common/uicr/Kconfig

```diff
@@ -19,6 +19,49 @@ config NRF_HALTIUM_UICR_PERIPHCONF
 	  to point at the blob. The initialization values are then loaded ahead of
 	  ahead of the application boot.
 
+config NRF_HALTIUM_UICR_RECOVERY
+	bool "Configure UICR RECOVERY field"
+	help
+	  Configure the UICR RECOVERY field to enable recovery firmware boot.
+	  This should be enabled by the main application to configure recovery
+	  firmware parameters. The recovery firmware itself does not need to
+	  enable this option.
```

Comment:
this should be at sysbuild level rather than adding it to every image whereby it's useless for most